query/test/functional/test_endpoints.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+import unittest
+
+import requests
+
+
+
+class TestQueryService(unittest.TestCase):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # self.batch = boto3.client('batch')
+        self.uri = None
+        # self.db_session_maker = DBSessionMaker()
+
+    def setUp(self):
+        self.deployment_stage = os.environ['DEPLOYMENT_STAGE']
+        self.api_url = f"https://{os.environ['API_HOST']}/v1"
```

Comment:
Since we have a config package, could you load these environment variables from there for consistency?

Reply:
👍